molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/core/entity/impl/GoNLAnnotator.java

```diff
@@ -110,7 +110,7 @@ public void init()
 				return createGoNlOutputAttributes();
 			}
 
-			public String postFixResource = "";
+			private String postFixResource = "";
```

Comment:
This should be a local variable, not a field

Reply:
I've tried to keep the fixes as minimal as possible and suggest to implement your comment separately since this code is quite hairy.

Reply:
@dennishendriksen is good. Bit worried that if we only fix the sonar warning and not the root cause, we'll not get the maximum benefit out of the warnings.